db/version_edit.cc

```diff
@@ -36,7 +36,8 @@ enum Tag : uint32_t {
   kNewFile = 7,
   // 8 was used for large value refs
   kPrevLogNumber = 9,
-  kMinLogNumberToKeep = 10,
+  kLastFlushSeqDecree = 10,
+  kMinLogNumberToKeep = 11,
```

Comment:
I suggest adding comment here to explain difference between pegasus version and official 6.6.4